lib/praxis/handlers/json.rb

```diff
@@ -8,12 +8,6 @@ class JSON
       # @raise [Praxis::Exceptions::InvalidConfiguration] if the handler is unsupported
       def initialize
         require 'oj'
-        begin
-          require 'json'
-        rescue LoadError # rubocop:disable Lint/SuppressedException
-        end
-        # Enable mimicing needs to be done after loading the JSON gem (if there)
-        ::Oj.mimic_JSON
```

Comment:
We can add this back to the spec helper to fix the specs.

As @blanquer mentioned, we should probably try to understand _why_ the specs are broken, i.e. why `JSON.dump` generates seemingly-invalid outputs when `oj` has been loaded.